spec/fixtures.rb

```diff
@@ -59,6 +59,11 @@ def self.included(base)
                 field.max 7 if field.respond_to?(:max)
                 field.step 1 if field.respond_to?(:step)
               end
+              action.field :validate_field do |field|
+                field.title 'validate field'
+                field.type :number
+                field.validation format: /^\d+$/, message: 'must be a number'
```

Comment:
Since it will be rendered into JSON, `format` should probably be a string. And without the anchors, as discussed, i.e. `'\d+'`.